tools/editor/property_editor.cpp

```diff
@@ -1864,6 +1867,15 @@ void CustomPropertyEditor::_modified(String p_string) {
 	updating=false;
 }
 
+void CustomPropertyEditor::_emit_changed_whole_or_field() {
+
+	if (!Input::get_singleton()->is_key_pressed(KEY_SHIFT)) {
```

Comment:
Well, if it isn't doable, ignore it -- maybe someone else would fix it one day :smiley: